### PR TITLE
Artifact Necklace Heirloom: Deeja-Wujeeta

### PR DESCRIPTION
**Pull request by Gotimanga**
## About the PR
Adds a new Heirloom for the character: Deeja-Wujeeta on the Umbra codebase. If this somehow uploads to the wrong repo I will cry.

## Why / Balance
Cool thing to own. This is a purely cosmetic heirloom. It has a custom artifact nodegraph with only one node - a node that on inspect will make everyone around it feel positive.

## Media

https://github.com/Umbra-Cosmatic-Drift/Umbra-CD/assets/127038462/7aa5c090-e1f0-45c3-a6c6-8c0f7c32ab75

Hopefully not too dark. I will change the video to a higher quality one if required.
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
Not required

### CODE REVIEW

File: Resources/Prototypes/_Umbra/Entities/Clothing/Neck/arti-necklace.yml
Comment (TsjipTsjip) [inline Resources/Prototypes/_Umbra/Entities/Clothing/Neck/arti-necklace.yml:1]: File should include a comment at the top of the entity block:
```yml
# Player: Deeja - Character: Deeja-Wujeeta
```

**Review by TsjipTsjip — CHANGES_REQUESTED** [review]

**Review by Aexxie — CHANGES_REQUESTED** [review]
You should rename the YAML file to `misc.yml`, and rename it internally to `ArtifactNecklace`. This includes file names, you should avoid using hyphens.

TL;DR
- Rename `arti-necklace.yml` to `misc.yml`
- Change the ID to be `ClothingNeckArtifactNecklace`
- Change the RSI to `artifactnecklace.rsi`

**Comment by Gotimanga** [comment]
@Aexxie I believe that's all the changes requested. I don't think I've missed anything

**Comment by Gotimanga** [comment]
Except for the fact I typed Misc with a K. Oops

**Review by Aexxie — APPROVED** [review]

**Review by TsjipTsjip — APPROVED** [review]
GM approval.

**Comment by Gotimanga** [comment]
:D

**Comment by Kukutis96513** [comment]
ooh nice
